flow/designs/asap7/mock-array/Element/config.mk

```diff
@@ -36,5 +36,9 @@ export MOCK_ARRAY_COLS        = $(word 2, $(MOCK_ARRAY_TABLE))
 
 # since we are specifying DETAILED_ROUTE_ARGS, we need to communicate the
 # same information to other stages in the flow.
-export MIN_ROUTING_LAYER = M2
-export MAX_ROUTING_LAYER = M5
+export MIN_ROUTING_LAYER      = M2
+export MAX_ROUTING_LAYER      = M5
+
+# make sure block level pins are not on the highest allowable routing layer
+export IO_PLACER_H            = M4
+export IO_PLACER_V            = M3
```

Comment:
Same question as before: how does this improve the results?

Reply:
Since power grid AND maximum routing layer for Element is M5, we should avoid having pins on M5. In this case, the obvious improvement is route-ability. With pins on M5, and smaller size, we have routing congestion. Having pins on M4 with the same size yield 0 routing error

Reply:
Is this generally the case? If so, any reason not to change the default behavior of the ASAP7 PDK?

Reply:
Yes it is. Nope ... The PDK was not targeting block level. Good idea to start ... Will add conditional in PDK